graylog2-web-interface/src/views/pages/NewDashboardPage.test.jsx

```diff
@@ -27,14 +27,11 @@ describe('NewDashboardPage', () => {
     jest.clearAllMocks();
   });
 
-  it('should render minimal', async () => {
+  it('should render minimal and show loading spinner initially', async () => {
```

Comment:
I would turn the description around:

```suggestion
  it('shows loading spinner before rendering page', async () => {
```